### PR TITLE
fix: deprecate isRainbowCurated for swap assets

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

fixes https://rainbowhaus.slack.com/archives/C02C8JQ313N/p1712086754141749

isRainbowCurated is deprecated so i updated it to use isVerified

### CODE REVIEW

File: src/entries/popup/hooks/useSearchCurrencyLists.ts [hunk 1]
```diff
@@ -170,6 +170,8 @@ export function useSearchCurrencyLists({
       fromChainId,
     });
 
+  console.log('blastVerifiedAssets', blastVerifiedAssets);
```
Comment (greg-schrammel) [inline src/entries/popup/hooks/useSearchCurrencyLists.ts:173]:
```suggestion

```

**Review by greg-schrammel — APPROVED** [review]

**Review by BrodyHughes — COMMENTED** [review]
This seemed to make the asset loading on the swaps screen lag really bad. Here is the performance now
<video src="https://github.com/rainbow-me/browser-extension/assets/41711440/5bd00f6e-41e7-4f72-a656-45e262cf9084" />

Vs Master:
<video src="https://github.com/rainbow-me/browser-extension/assets/41711440/a51feb03-c670-4d58-8b37-8aae3b39b215" />

**Comment by estebanmino** [comment]
@BrodyHughes i just added a default swap gas value that i think was causing that issue

**Comment by estebanmino** [comment]
ah no is not related

**Review by DanielSinclair — APPROVED** [review]
There seems to be performance impact when calling `getVerifiedAssets` and filtering the whole `verifiedAssets` list (or returning that lengthy list). This list is already filtered on `isVerified` at the API request, so I instead temporarily limited the number of results to 50 which brought back performance a bit

**Pull request merged**